src/modules/schema.js

```diff
@@ -27,6 +27,24 @@ export default class Schema {
     this.id = id || createNewDockBlobId();
   }
 
+  static fromJSON(json) {
+    const {
+      id, schema, author, ...rest
```

Comment:
The only extra property is `signature` here, correct? Would we want a schema object with additional properties? If not then we should only check for the ones we know about and skip others.

Reply:
yeah and we omit signature in our tojson method. good point here, have updated to remove need for rest